src/client/api/quoteBundleSelectors.ts

```diff
@@ -70,19 +70,15 @@ export const isYouthOffer = (bundle: QuoteBundle): boolean => {
     .every((quote) => quote.data.isYouth === true)
 }
 
-const HOME_HOUSE_INSURANCE_TYPES: Array<InsuranceType> = [
-  InsuranceType.DANISH_HOME_CONTENT,
-  InsuranceType.NORWEGIAN_HOME_CONTENT,
-  InsuranceType.NORWEGIAN_HOUSE,
-  InsuranceType.SWEDISH_APARTMENT,
-  InsuranceType.SWEDISH_HOUSE,
-]
-
 export const getMainQuote = (bundle: QuoteBundle) => {
   if (isMultiQuote(bundle)) {
-    const mainQuoteInBundle = bundle.quotes.find((quote) =>
-      HOME_HOUSE_INSURANCE_TYPES.includes(quote.data.type),
+    const houseQuote = bundle.quotes.find((quote) =>
+      quoteSelector.isHouse(quote),
+    )
+    const homeContentsQuote = bundle.quotes.find((quote) =>
+      quoteSelector.isHomeContents(quote),
     )
+    const mainQuoteInBundle = houseQuote || homeContentsQuote
```

Comment:
Does this work with Car?

Reply:
Not sure what this is used for, I don't think I've used the concept of main quote myself, but perhaps Marian has? Try it out with quote `e99c3e62-fee1-4824-b802-20ba80b61bfc`

Reply:
Well, this did not work with Car in the first place right?

Reply:
For future reference it can be good to know that car offers are not `isMultiQuote` so for them it will just `return bundle.quotes[0]`